src/typeahead/typeahead-window.ts

```diff
@@ -39,6 +41,12 @@ export class NgbTypeaheadWindow implements OnInit {
   activeIdx = 0;
 
   /**
+   *  An optional id for the typeahead. The id should be unique.
+   *  If not provided, it will be auto-generated.
+   */
+  @HostBinding('attr.id') @Input() id: string = undefined;
```

Comment:
Could you please use 'host' instead of `@HostBinding` so all the host bindings are in one place only?